QUEST.tsv

```diff
@@ -522,210 +522,210 @@ QUEST_20150317_000521	Seems to be asleep.
 QUEST_20150317_000676	The Report of the High Barrier Magic
-QUEST_20150317_000677	The barrier which is in this tower can not be created by the demon lord himself.
-QUEST_20150317_000678	It is like a net which is connected tightly, {nl}so one with a normal level of divine power can not escape from it.
-QUEST_20150317_000679	For what reason was this barrier created? {nl}Is it related to the fact the goddess who controls the flames of this world is staying here?
-QUEST_20150317_000680	Agailla Flurry called Gabija to the tower, so this barrier is not meant to lock Gabija in. {nl}To find out the reason for this, we first have to understand the composition of the barrier.
-QUEST_20150317_000681	After a long investigation, we found out that this barrier was {nl}not only created due to the power of Helgasercle, but it was also mixed with other kinds of mana.
-QUEST_20150317_000682	And we also found out that mana was composed by aligning the foundation stones in a position.
-QUEST_20150317_000683	We were able to retrieve some of those foundation stones from various areas in the kingdom. {nl}The new information that we found here was astonishing.
-QUEST_20150317_000684	The mana provided to use barrier magic wasn't voluntary. {nl}We only had to predict that the levels of the demon lords were divided and sealed into the foundation stones and aligned into a position.
-QUEST_20150317_000685	The key to unlock this barrier is in this foundation stone. {nl}We will find the answer after we block the invasion of Helgasercle.
+QUEST_20150317_000677	The barrier in this tower could not have possibly been created by the demon lord.
+QUEST_20150317_000678	It is like a tight-knit net. {nl}Someone with a normal level of divine power cannot escape from it.
+QUEST_20150317_000679	For what reason was this barrier created?{nl}Is it related to the fact that the Goddess who controls the flames of this world is staying here?
+QUEST_20150317_000680	Agailla Flurry called Gabija to this tower, so this barrier is not meant to lock Gabija in. {nl}To find out the reason for this, we first have to understand the composition of the barrier.
+QUEST_20150317_000681	After a long investigation, we've found that this barrier was {nl}not only created by the power of Helgasercle, but also with other kinds of mana.
+QUEST_20150317_000682	We've also found that mana was composed by aligning the foundation stones in a certain position.
+QUEST_20150317_000683	We were able to retrieve some of those foundation stones from various areas in the kingdom. {nl}The new information that we found here with them was astonishing.
+QUEST_20150317_000684	The mana provided to use barrier magic wasn't voluntary. {nl}We theorize that the levels of the demon lords were divided and sealed into the foundation stones and aligned into a position.
+QUEST_20150317_000685	The key to unlock this barrier is in the foundation. {nl}We'll continue to find the answer after we block Helgasercle's invasion.
```

Comment:
Changing the second sentence changes the meaning from "surely finding an answer after completing a task" to "continuing to find more information after completing a task".  If that is correct considering the context of the situation ( it looks like it is ), then I'd just say that the wording should be changed a little.
"We'll continue to find more answers"
or
"We'll continue to search for the answer"